### PR TITLE
Improve Pagination & Fix slider arrow position

### LINKED ISSUE #42: Improve Pagination component
### Explanation
 The pagination component is presenting some bugs in small screens (check screenshots). I propose three solutions to solve these problems:
 
 - To maintain the logic simply, I propose changing the functionality of the pagination, instead of showing multiple numbers, only show three numbers and the left and right arrows. Some like this: "< 1 **2** 3 >" 
 - To reduce the effort you can use a library.I propose to use [React Paginate](https://www.npmjs.com/package/react-paginate) that looks very similar to our design
 - You can continue working on the component that we have and try to do the functionality shown at the design, this looks like a very difficult work.

### Screenshots
As you see, the pagination is overflow when the screen is small.
imagen.

Design screenshot
imagen.

### Checklist
 This issue can be closed when the following tasks are completed:
 
 - [ ] You have chosen one of the three solutions proposed and started to work on it! Please, let me know in the comments what solution you have chosen and the estimation of time that you think this will take for you.
 - [ ] The pagination is working correctly and looks good on small screens. 
 - [ ] It has visual styles similar to the design.
 
### References
- https://www.figma.com/file/zzEwuPfGTiZh4EA11MyZ5c/Sencico?node-id=0%3A1
- https://www.npmjs.com/package/react-paginate

### LINKED ISSUE #43: Fix slider arrow position in small screens
### Explanation
On small screens the slider arrows look unaligned (check screenshots), this slider use the library React Slick, you can check the documentation here: https://react-slick.neostack.com/ . This library allow the use padding and other CSS properties over their parts. To fix this problem please check if something is affecting the arrows and check how you can fix it using CSS and the documentation.

### Screenshots
imagen

### Checklist
 This issue can be closed when the following tasks are completed:
 
 - [ ] The arrows are centered in small screens.
 
### References
- https://react-slick.neostack.com/

### PR DESCRIPTION
**Pull request by Eghinner**
### Description

- Implement [react-paginate](https://www.npmjs.com/package/react-paginate ) as @Katsu08 proposed to look like design.

- In Slider component use the powerful "truncate" of Tailwind to avoid overflow

- A little change in index.css to fix background color of active page

 #Fixes #42 
 #Fixes #43

#### Screenshots
screenshot-localhost_3000-2022 02 25-23_11_24
screenshot-localhost_3000-2022 02 22-16_34_35

### CODE REVIEW

**Review by heliomar-pena — COMMENTED** [review]
I have checked the functionality and it's working perfect!

imagen
imagen
imagen

I'm checking the code

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "query-string": "7.0.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-paginate": "^8.1.2",
```
Comment (heliomar-pena) [inline package.json]: Please freeze package

```suggestion
    "react-paginate": "8.1.2",
```

File: src/components/Slider.js [hunk 2]
```diff
@@ -95,13 +95,14 @@ const ProductSlider = ({ data, categories, itemClick, className, title }) => {
               onClick={(e) => itemClick(e, item.id, categories)}
             >
               <div
+                title={item.name}
```
Comment (heliomar-pena) [inline src/components/Slider.js]: Sorry, I don't understand what is the "title" attribute in div?

**Review by heliomar-pena — APPROVED** [review]
Good work improving the pagination!

**Review by e-azocar — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

**Comment by angelik0828** [comment]
in functionality all very well I do not see the rounded tips in the elements I leave you this guide image
image

**Review by Israel-Laguan — APPROVED** [review]

File: src/components/Slider.js (hunk 2 above)
Comment (Israel-Laguan) [reply src/components/Slider.js]: Maybe you meant `id` or `key`?

Comment (Eghinner) [reply src/components/Slider.js]: I just wanted to show full name with a "title" since ellipsis cuts it

**Comment by Israel-Laguan** [comment]
Please rebase

**Comment by heliomar-pena** [comment]
Good work @Eghinner ! Please rebase and we could merge the pull request

**Comment by Israel-Laguan** [comment]
@Katsu08 please check, seems like can be merged now

**Pull request merged**